### DISCUSSION TITLE
New pattern for command's

### DISCUSSION THREAD

**Discussion by alix1383** [laravel/framework#51256] [Ideas]
I think it is a good idea to change the pattern of new commands.
 For example, currently if you want to create a new command with `DoFoo` name, you would have something like `app:do-foo`.
I suggest replacing `-` with `:`

Final results is : `app:do:foo`

**Comment by ahinkle** [comment]
Popular Laravel packages (like Filament) use the dash. I'm not aware of many that use the colon.

**Comment by mbabker** [comment]
The colon has semantic meaning in the Symfony console (which Laravel builds upon) and is used to create namespaces for command names.  So arbitrarily changing hyphens with colons implies a bit more of a change in the console application than just what character you type in.
